### DISCUSSION TITLE
multi pool beta feature

### DISCUSSION THREAD

**Discussion by wibed** [cilium/cilium#36248] [Q&A] [answered]
what are the upsides of the multi pool beta feature against using a classic loadbalancer

**Comment by joestringer** [comment, marked as answer]
[Multi-pool IPAM](https://docs.cilium.io/en/stable/network/concepts/ipam/multi-pool) allows Pod IP addresses to be allocated from multiple ranges, making it easier to assign IPs based on attributes of Pods and to dynamically scale the number of IPs that are made available.

**Comment by wibed** [comment]
what i had a problem to understand. is if i had multipool addresses how can i allocate them within the definition of my
load balancer?
